test/images/README.md

```diff
@@ -182,7 +182,8 @@ After all the above has been done, run the [desired tests](https://github.com/ku
 
 Now that you have made the changes and tested locally, you are ready to share those changes.  This is a multi step process:
 
-1. Open a pull request with your changes to the test image (the new functionality and version bump). Go through the review process and merge the pull request. See this [pull request](https://github.com/kubernetes/kubernetes/pull/99860/files) for an example.
+1. In the same pull request containing your proposed changes, bump the version of the image in question. Each test image has
+a VERSION file in its directory. See [this pull request](https://github.com/kubernetes/kubernetes/pull/108771) for an example.
```

Comment:
```suggestion
a VERSION file in its directory. See [this pull request](https://github.com/kubernetes/kubernetes/pull/99860) for an example.
```

your PR only bumps the version, the previous one has the version + the changes, I think is better to keep the reference to the previous one

Reply:
The previous PR updates `build/dependencies.yaml` which is no longer needed though

Reply:
(also okay to not include a reference PR which the steps ahve just been reduced to "bump the VERSION file"

Reply:
okay I removed the reference PR and just gave an example of where the VERSION path is (e.g. test/images/agnhost/VERSION)

Reply:
ohh, I didn't knew sorry